models/tridimensional/data_kleinstiver/cas9_mutants_stats.py

```diff
@@ -137,3 +137,106 @@ def main():
 
 if __name__ == '__main__':
     main()
+
+    
+def idx_freq(pam):
+    '''
+    Given a pam site returns the indices of mutation and frequency of mutation
+    occurring at said indices
+    '''
+    
+    mut_lst = []
+    for mutant in mutants_kleinstiver:
+        if mutant['pam'] == pam:
+            mut_lst.append(mutant)
+    id_dict = {}
+    for mutant in mut_lst:
+        for mutation in mutant['mutations']:
+            idx = mutation['aa_idx']
+            if idx in id_dict.keys():
+                id_dict[idx] = id_dict[idx] + 1
+            if not idx in id_dict.keys():
+                id_dict[idx] = 1
+    return id_dict
+
+def ss_freq(pam):
```

Comment:
This duplicates the function of find_ss_counts above almost exactly The only difference I can see is that find_ss_counts has keys for every secondary structure element (i.e. includes ones that are not mutated), unless I am very confused?